lib/secure-boot-lib.robot

```diff
@@ -218,3 +218,26 @@ Execute File In UEFI Shell
     Press Enter
     ${out}=    Read From Terminal Until    FS0:\\>
     RETURN    ${out}
+
+Remove Old Secure Boot Keys
+    [Documentation]    Removes all files and directories in
+    ...    `/usr/share/secureboot/keys/`
+    Execute Linux Command Without Output   rm -rf /usr/share/secureboot/keys/*
+
+Generate Secure Boot Keys
+    [Documentation]    Generates new Secure Boot keys. Returns true if succeeded.
+    ${out}=    Execute Linux Command    sbctl create-keys
+    ${sb_status}=    Run Keyword And Return Status
+    ...    Should Contain    ${out}    Secure boot keys created!
+    RETURN    ${sb_status}
+
+Enroll Secure Boot Keys
+    [Documentation]    Enrolls current keys to EFI. Returns true if succeeded.
+    ${out}=    Execute Linux Command    sbctl enroll-keys --yes-this-might-brick-my-machine
+    ${sb_status}=    Run Keyword And Return Status
+    ...    Should Contain    ${out}    Enrolled keys to the EFI variables!
+    RETURN    ${sb_status}
+
+Sign All Boot Components
+    [Documentation]    Signs boot components with current keys
+    Execute Linux Command    sbctl verify | awk -F ' ' '{print $2}' | tail -n+2 | xargs -I "#" sbctl sign "#"
```

Comment:
@m-iwanicki it will try to sign even those components that are already signed?

Reply:
Yes. I just checked and it will sign multiple times each time increasing it's size.
It's possible to delete signature with `sudo sbattach --signum 1 --remove /path/to/signed/efi/file`
```
# sudo sbctl sign /boot/efi/EFI/BOOT/BOOTX64.EFI
Signed /boot/efi/EFI/BOOT/BOOTX64.EFI
# sudo sbattach --signum 1 --remove /boot/efi/EFI/BOOT/BOOTX64.EFI
warning: data remaining[827536 vs 953088]: gaps between PE/COFF sections?
```
But size is a little bigger than before signing so it's not perfect.

Reply:
If we are sure that every file will be signed then I am ok with that for this test.